README.md

```diff
@@ -412,13 +412,26 @@ If it becomes necessary to exempt a namespace from Gatekeeper entirely (e.g. you
 
 > NOTE: Verbose logging with DEBUG level can be turned on with `--log-level=DEBUG`.  By default, the `--log-level` flag is set to minimum log level `INFO`. Acceptable values for minimum log level are [`DEBUG`, `INFO`, `WARNING`, `ERROR`]. In production, this flag should not be set to `DEBUG`.
 
+Verbose logging specific to pod readiness tracking can be enabled using the `statsEnabled` flag of the Config resource:
```

Comment:
Should we add this to the readme _after_ we cut a new release that includes this change?

Reply:
@ritazh I'm fine either way, let me know what you prefer.

Reply:
Thanks for the PR @shomron! My preference is update the readme _after_ the feature is part of in a release in case people use it with the latest release and the flag is not part of any release yet.

Reply:
Sure, I'll take it out. Do we need a staging area for documentation changes to match staging for manifests? Or should we park these out-of-band?

Reply:
Updated. Parked in the new `staging_docs/` directory.